File: SDUTY/Sduty_Server/src/main/java/com/d108/sduty/dto/Timeline.java
package com.d108.sduty.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@ToString
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class Timeline {
	private Profile profile ;
	private Story story ;
	private int cntReply;
	private List<Reply> replies;
	private int numLikes;
	private boolean likes;
	private boolean scrap;
	private JobHashtag jobHashtag;
	private List<InterestHashtag> interestHashtags;
	
	
}

File: SDUTY/Sduty_User/app/src/main/java/com/d108/sduty/model/dto/Timeline.kt
package com.d108.sduty.model.dto

data class Timeline(var profile: Profile, var story: Story, var cntReply: Int, var replies: MutableList<Reply>, var numLikes: Int, var likes: Boolean, var scrap: Boolean, var jobHashtag: JobHashtag, var interestHashtags: List<InterestHashtag>) {
}